src/index.js

```diff
@@ -13,9 +20,31 @@ export const storybook = (opts = { }) => {
     staticDir,
     storyWrapper = './default-wrapper',
     autofill = false,
+    debug = false,
     stories
   } = opts
 
+  const storybookVersion = findPkgUpDep('@storybook/react')
+  if (!storybookVersion) {
+    console.error('[docz-plugin-storybook] "@storybook/react" is a required peer dependency. Aborting.')
+    process.exit(1)
+  }
+
+  if (!semver.satisfies(storybookVersion, minStorybookSemver)) {
+    console.error(`[docz-plugin-storybook] "@storybook/react" unsupported version v${storybookVersion}. Requires "${minStorybookSemver}". Aborting.`)
+    process.exit(1)
+  }
+
+  const isStorybookV3 = semver.satisfies(storybookVersion, v3StorybookSemver)
+  const isStorybookV4 = semver.satisfies(storybookVersion, v4StorybookSemver)
+
+  if (isStorybookV3 === isStorybookV4) {
```

Comment:
(Silvi on Travis' laptop on flight to London - poor Kevin missed his connection. we love you kevin)

This is legitness